src/components/Queue.vue

```diff
@@ -83,7 +83,7 @@ export default {
                 // so that we can reflect any changes in the Queue component.
                 var queueState = val.sendingStatus;
                 if(!queueState && val.sentData) {
-                    var message = this.genQueueSuccessMessage(val.sentData);
+                    var message = this.queueSuccessMessage(val.sentData);
```

Comment:
Nope but we are doing a refactor. So taking one step at a time. These are the final hours!